lib/middleware/ContentMiddleware.js

```diff
@@ -0,0 +1,34 @@
+const { relative, sep } = require('path');
+
+const statik = require('@shellscape/koa-static/legacy');
```

Comment:
typo `statik` -> `static`

Reply:
hah that's not actually a typo because our linting forbids using reserved words for variable names

Reply:
mmmmmm nah 😄